### DISCUSSION TITLE
Tag Ec2 resource with env key, if it not contains the values [prod,dev,stage]

### DISCUSSION THREAD

**Discussion by sdobhal369** [cloud-custodian/cloud-custodian#7469] [AWS]
Hi all, 
 I got stuck while implementing policy.yml for ec2 resource. My problem is that I have to check the env key value on Ec2, if it is there then I have to validate it. Like env key must have any value from [prod, dev, stage] not any other value.

**Comment by thisisshi** [comment]
By env key are you referring to an environment variable?
